recipes/spades/meta.yaml

```diff
@@ -1,22 +1,45 @@
 {% set version = "3.13.1" %}
+{% set sha256 = "8da29b72fb56170dd39e3a8ea5074071a8fa63b29346874010b8d293c2f72a3e" %}
 
 package:
   name: spades
   version: {{ version }}
 
 source:
-  # We currently cannot build SPAdes from source, because it requires g++ 5.3.1 or later,
-  # which bioconda does not yet support.  Therefore, we fetch the statically compiled binaries.
-  url: https://github.com/ablab/spades/releases/download/v{{ version }}/SPAdes-{{ version }}-Darwin.tar.gz  # [osx]
-  sha256: 8898d93688ede7dd82048917aae102c6b0b9b0cdbff0e4a1ee7f8052fb17e955  # [osx]
-  url: https://github.com/ablab/spades/releases/download/v{{ version }}/SPAdes-{{ version }}-Linux.tar.gz  # [linux]
-  sha256: 3a342ab10087e0faaa2efe2fb6dfdf1774f26c7ef5087b7f868d8a85ebd5ef18  # [linux]
+  url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-change-verbosity.patch
+    - 0002-disable-parallel-libstdc.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libcxx # [osx]
```

Comment:
Does this really need libcxx?

Reply:
Actually, I think `{{ compiler('cxx') }}` pulls that in. I'll play around with this a bit and revert & merge if I break something :)